tensorflow/lite/kernels/elementwise.cc

```diff
@@ -463,7 +585,8 @@ TfLiteStatus SquareEval(TfLiteContext* context, TfLiteNode* node) {
                status);
   }
 #endif  // TFLITE_KERNEL_USE_XNNPACK
-  return EvalNumeric(context, node, [](float f) { return f * f; });
```

Comment:
Looks like FP16 and BF1G dtype support has been missed in SquareEval. Or am I missing something?

Reply:
Hi Vamsi, the square op was not part of the list of ops given to us, and hence the support wasn't added

Reply:
In that case this line should be fixed-
`GENERIC_PREPARE(PrepareSquare, elementwise::IsNumericSupportedType, "Square")`, because `IsNumericSupportedType` has been touched in the PR and no longer applies to what SquareOp supports.

If its quick, I suggest extending the extra dtype support to Square as well. Thank you.